src/style/colors.ts

```diff
@@ -203,7 +203,15 @@ function validateHex(value: string) {
   if (!/^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/.test(value)) {
     throw new Error(`${value} is invalid hex color`);
   }
-  return value.replace('#', '');
+  value = value.replace('#', '');
+  if (value.length === 3) {
+    value = threeDigitHexToSix(value);
```

Comment:
I know using regex to duplicate the string look fancy (: 
But maybe a more readable approach will be to concatenate value with itself.